components/match2/wikis/apexlegends/match_summary.lua

```diff
@@ -0,0 +1,679 @@
+		},
+	},
+	{
+		class = 'cell--team',
+		iconClass = 'fas fa-users',
+		header = {
+			value = 'Team',
+		},
+		row = {
+			value = function (opponent)
+				return CustomMatchSummary._displayOpponent(opponent)
+			end,
+		},
+	},
+	{
+		class = 'cell--total-points',
+		iconClass = 'fas fa-star',
+		header = {
+			value = 'Total Points',
+		},
+		row = {
+			value = function (opponent)
+				return opponent.score
+			end,
+		},
+	},
+	game = {
+		{
+			class = 'panel-table__cell__game-placement',
+			iconClass = 'fas fa-trophy-alt',
+			header = {
+				value = 'P',
+			},
+			row = {
+				class = function (opponent)
+					return PLACEMENT_BG[opponent.placement]
+				end,
+				value = function (opponent)
+					local icon, color = CustomMatchSummary._getIcon(opponent.placement)
+					return mw.html.create()
+							:tag('i'):addClass('panel-table__cell-icon'):addClass(icon):addClass(color):done()
+							:tag('span'):addClass('panel-table__cell-game__text')
+									:wikitext(CustomMatchSummary._displayRank(opponent.placement)):done()
+				end,
+			},
+		},
+		{
+			class = 'panel-table__cell__game-kills',
+			iconClass = 'fas fa-skull',
+			header = {
+				value = 'K',
+			},
+			row = {
+				value = function (opponent)
+					return opponent.scoreBreakdown.kills
+				end,
+			},
+		},
+	}
+}
+
+local GAME_STANDINGS_COLUMNS = {
+	{
+		class = 'cell--button',
+		header = {
+			value = '',
+		},
+		row = {
+			value = function (opponent)
+				return ''
+			end,
+		},
+	},
+	{
+		class = 'cell--rank',
+		iconClass = 'fas fa-hashtag',
+		header = {
+			value = 'Rank',
+		},
+		row = {
+			value = function (opponent)
+				local icon, color = CustomMatchSummary._getIcon(opponent.placement)
+				return mw.html.create()
+						:tag('i'):addClass('panel-table__cell-icon'):addClass(icon):addClass(color):done()
+						:tag('span'):wikitext(CustomMatchSummary._displayRank(opponent.placement)):done()
+			end,
+		},
+	},
+	{
+		class = 'cell--team',
+		iconClass = 'fas fa-users',
+		header = {
+			value = 'Team',
+		},
+		row = {
+			value = function (opponent)
+				return CustomMatchSummary._displayOpponent(opponent)
+			end,
+		},
+	},
+	{
+		class = 'cell--total-points',
+		iconClass = 'fas fa-star',
+		header = {
+			value = 'Total Points',
+		},
+		row = {
+			value = function (opponent)
+				return opponent.score
+			end,
+		},
+	},
+	{
+		class = 'cell--placements',
+		iconClass = 'fas fa-trophy-alt',
+		header = {
+			value = 'Placement Points',
+		},
+		row = {
+			value = function (opponent)
+				return opponent.scoreBreakdown.placePoints
+			end,
+		},
+	},
+	{
+		class = 'cell--kills',
+		iconClass = 'fas fa-skull',
+		header = {
+			value = 'Kill Points',
+		},
+		row = {
+			value = function (opponent)
+				return opponent.scoreBreakdown.killPoints
+			end,
+		},
+	},
+}
+
+---@param args table
+---@return string
+function CustomMatchSummary.getByMatchId(args)
+	local match = MatchGroupUtil.fetchMatchForBracketDisplay(args.bracketId, args.matchId)
```

Comment:
do we potentionally ever have reset match being a thing?

Reply:
Nop, can't see that working